packages/next-templates/src/app/meldingen/graffiti-melding/form/page.tsx

```diff
@@ -66,182 +62,185 @@ export default function Home() {
+                  <UnorderedListItem>Bestanden moeten kleiner zijn dan 10 MB.</UnorderedListItem>
+                  <UnorderedListItem>Toegestane bestandstypen: gif, jpg, jpeg, png.</UnorderedListItem>
+                </UnorderedList>
+                <div className="voorbeeld-container-bijlage--flex">
+                  <Button appearance="secondary-action-button">Bestand kiezen</Button>
+                  <Paragraph className="voorbeeld-paragraph-bijlage">Geen bestand gekozen</Paragraph>
+                </div>
+              </section>
+              <section className="voorbeeld-section--spacing">
+                <Heading2>Op welke locatie heeft de melding betrekking?</Heading2>
+                <FormFieldTextbox
+                  id={`place-${placeField.name}`}
+                  className="voorbeeld-textbox"
+                  invalid={!!errors[placeField.name]}
+                  errorMessage={String(errors[placeField.name]?.message)}
+                  aria-label="Plaats"
+                  label="Plaats"
+                  {...placeField}
+                />
+                <FormFieldTextbox
+                  id={`street-${streetField.name}`}
+                  className="voorbeeld-textbox"
+                  invalid={!!errors[streetField.name]}
+                  errorMessage={String(errors[streetField.name]?.message)}
+                  aria-label="Straat"
+                  label="Straat"
+                  {...streetField}
+                />
+                <FormFieldTextbox
+                  id={`height-${heightField.name}`}
+                  className="voorbeeld-textbox"
+                  invalid={!!errors[heightField.name]}
+                  description="Bijvoorbeeld het huisnummer, parkeerplaats of speeltuintje."
+                  errorMessage={String(errors[heightField.name]?.message)}
+                  aria-label="Ter hoogte van"
+                  label="Ter hoogte van"
+                  {...heightField}
+                />
+              </section>
+              <section className="voorbeeld-section--spacing">
+                <Heading2>Hoe kunnen we u bereiken voor meer informatie?</Heading2>
+                <Fieldset role="radiogroup">
+                  <FieldsetLegend>Anoniem Melden?</FieldsetLegend>
+                  <FormFieldDescription className="voorbeeld-formField-description--spacing">
+                    Als u iets anoniem meld kunnen we u niet informeren over het verloop van de afhandeling voor deze
+                    melding.
+                  </FormFieldDescription>
+                  <FormField className="voorbeeld-formField" type="radio">
+                    <FormLabel className="voorbeeld-form-label" htmlFor="anoniem-ja" type="radio">
+                      Ja
+                    </FormLabel>
+                    <RadioButton className="voorbeeld-form-field__input" id="anoniem-ja" name="anoniem" value="1" />
```

Comment:
dit kan voorbeeld-radio-button zijn als je hebt besloten om het met of zonder camelcasing classnames gaat doen